package.json

```diff
@@ -30,13 +30,16 @@
     "axios": "^0.26.0",
     "base64url": "^3.0.1",
     "bullmq": "^3.5.4",
+    "canvas": "^2.11.2",
     "class-variance-authority": "^0.6.1",
     "cookie-parser": "^1.4.6",
     "cors": "^2.8.5",
     "date-fns": "^2.28.0",
     "dotenv": "^16.0.0",
     "ejs": "^3.1.7",
+    "eslint-config-prettier": "^9.0.0",
     "express": "^4.17.2",
+    "finnish-bank-utils": "git+https://github.com/vkomulai/finnish-bank-utils.git",
```

Comment:
Has to be GitHub as latest version is not published to npm

Reply:
Can this be pinned to a specific tag or commit?

Reply:
It can be pinned to a specific commit.

- [ ] Pin to the most recent commit